engine/shared/map/init.lua

```diff
@@ -540,9 +529,6 @@ function map:loadLayers( layers )
 		local layer = map.layer( layerData )
 		layer:setMap( self )
 		layer:parse()
-
-		local tileset = self:getTileset( layer )
```

Comment:
This doesn't make sense anymore since Tiled provides support for multiple tilesets in a single layer, right?

Reply:
Thats correct. I opted to delete `map:getTileset(layer)` because `layer`s can just pull the loaded tilesets directly from the owning `map`. An accessor function might help pretty up the API though.